rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -273,6 +273,20 @@ public String editNodeSource(String sessionId, String nodeSourceName, String inf
                                                                                            nodesRecoverable));
     }
 
+    @Override
+    public String updateDynamicParameters(String sessionId, String nodeSourceName, String infrastructureType,
+            String[] infrastructureParameters, String[] infrastructureFileParameters, String policyType,
+            String[] policyParameters, String[] policyFileParameters) throws RestServerException, ServiceException {
+        return executeFunctionReturnStreamAsString(restClient -> restClient.updateDynamicParameters(sessionId,
+                                                                                                    nodeSourceName,
+                                                                                                    infrastructureType,
+                                                                                                    infrastructureParameters,
+                                                                                                    infrastructureFileParameters,
+                                                                                                    policyType,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)